modules/psap-driver-toolkit-pulling.adoc

```diff
@@ -28,7 +28,11 @@ The driver-toolkit image for the latest minor release will be tagged with the mi
 +
 [source,terminal]
 ----
-$ oc adm release info 4.12.0 --image-for=driver-toolkit
+# For x86 image:
```

Comment:
Wondering if we should add a not about specifying the release, and expecialy about the `.0` , which will evolve with the lifetime of 4.12 . 
Something like: _Update the `.0` with the current version of your cluster._

Reply:
I opted for a variable we use, `{product-version}`, which fills in the viewed version (for example, 4.12, 4.13, and so on). I also just made it `.z` so the user knows to fill in the rest. Thanks!